src/main/webapp/app/shared/userMessage/BetaSiteMessage.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import * as styles from './styles.module.scss';
+import { Container } from 'reactstrap';
+import { BANNER_HEIGHT } from 'app/stores/layout.store';
+import { IRootStore } from 'app/stores';
+import { componentInject } from '../util/typed-inject';
+import { observer } from 'mobx-react';
+
+const BetaSiteMessage = (props: StoreProps) => {
```

Comment:
Should this component be called `Banner` and have the message as pass in component which is BetaSiteMessage?

Reply:
Thought about making it general, but seems like this is the only use case right now. Called it specifically BetaSiteMessage because it has some logic to check if is beta site

Reply:
Sounds good. Good to merge if you could take a look at the test.